src/ProfilesPage/ProfilePage.js

```diff
@@ -162,13 +176,19 @@ class ProfilePage extends Component {
 
   deleteUserButton() {
     const admin = sessionStorage.getItem("isAdmin");
-    return (
-      <a href="/message">
-        <button className="btn btn-danger" onClick={this.onDelete.bind(this)}>
-          Delete User
-        </button>
-      </a>
-    );
+    const superuser = this.props.userInfo.superuser;
+    
+    console.log(superuser);
+    {/*if superuser's value is false render deleleteUser button.*/}
+    if(superuser === false) {
+      return (
+        <a href="/message">
+          <button className="btn btn-danger" onClick={this.onDelete.bind(this)}>
+            Delete User
+          </button>
+        </a>
```

Comment:
I'm having trouble understanding the `button` wrapped in an `a`?

Reply:
not sure why that was like that but it is fixed.